dask_planner/src/sql.rs

```diff
@@ -195,7 +199,67 @@ impl ContextProvider for DaskSQLContext {
         None
     }
 
-    fn get_aggregate_meta(&self, _name: &str) -> Option<Arc<AggregateUDF>> {
+    fn get_aggregate_meta(&self, name: &str) -> Option<Arc<AggregateUDF>> {
+        let acc: AccumulatorFunctionImplementation =
+            Arc::new(|| Err(DataFusionError::NotImplemented("".to_string())));
```

Comment:
```suggestion
            Arc::new(|_| Err(DataFusionError::NotImplemented("".to_string())));
```

Reply:
I got `Arc::new(|| Err(DataFusionError::NotImplemented("".to_string())));` from #698 and it looks like it expects 0 arguments, so I'm going to revert this change.